examples/hashable_enum.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Zlikavac32\Rick\Examples;
+
+use Ds\Hashable;
+use Ds\Set;
+use Zlikavac32\Enum\Enum;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+if (!extension_loaded('ds')) {
```

Comment:
The extension is not strictly necessary for this example; the polyfill works as well. So instead of checking if the extension is loaded, you can check if class Ds\Hashable exists.

Reply:
@mjpvandenberg I think they had plan to ditch polyfill, so this would work in the long run. Also, i don't think that polyfill makes sense in the context of this extension.

Reason why it exists is to avoid using arrays for everything

Reply:
Indeed, the author is considering to ditch the polyfill. My suggested change here would work in both situations (polyfill as well as C extension), making the example a bit more accessible, is all.